test/integration/targets/docker_network/tasks/tests/options.yml

```diff
@@ -92,3 +92,93 @@
     - driver_options_3 is not changed
     - driver_options_4 is changed
     - driver_options_5 is not changed
+
+####################################################################
+## scope ###########################################################
+####################################################################
+
+- block:
+  - name: scope
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: bridge
+      scope: local
+    register: scope_1
+
+  - name: scope (idempotency)
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: bridge
+      scope: local
+    register: scope_2
+
+  - name: swarm
+    docker_swarm:
+      state: present
+      advertise_addr: "{{ansible_default_ipv4.address}}"
+
+  - name: scope (change)
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: overlay
```

Comment:
If there's a better way to do this, please let me know. I don't think docker includes any drivers that support multiple modes (or at least, I wasn't able to figure out how to get any of them to switch in a way that was reflected in the output - swarm and global both report as swarm for some reason).

Reply:
I think you should mention that as a comment in the test, so it's clear that this is not an oversight, but an conscious (unfortunate) decision you had to make.